### PR TITLE
Update connector documentation

### LINKED ISSUE #963: Documentation for new connector API

### PR DESCRIPTION
**Pull request by aklenik**
- Write detailed docs about how to implement connectors
- Update connector-specific pages based on the new API
- Fix minor inconsistencies in other pages

Live preview: https://aklenik.github.io/caliper/

Relates to #1006 
Closes #963 

Signed-off-by: Attila Klenik <a.klenik@gmail.com>

### CODE REVIEW

**Review by nklincoln — CHANGES_REQUESTED** [review]
two minor comments 👍

File: docs/vNext/Fabric_Configuration.md [hunk 1]
```diff
@@ -132,58 +113,24 @@ are specified then targetPeers will take precedence
...
+    invokerIdentity: 'client0.org2.example.com',
+    timeout: 10
 };
 
-return this.sutAdapter.querySmartContract('marbles', '', settings, 3);
+await this.sutAdapter.sendRequests(requestSettings);
 ```
 
-`querySmartContract` also accepts an array of `querySettings` as the second arguments. However, Fabric does not support submitting an atomic batch of queries, so there is no guarantee that their execution order will remain the same (although it's highly probably, since queries have shorter life-cycles than transactions).
+> __Note:__ `sendRequests` also accepts an array of request settings. However, Fabric does not support submitting an atomic batch of transactions like Sawtooth, so there is no guarantee that the order of these transactions will remain the same, or whether they will reside in the same block.
```
[the remark this answers is not in the document]
Comment (aklenik) [reply docs/vNext/Fabric_Configuration.md:128]: It's just a typical example of tx batches that can declare dependencies between txs in the same batch. Just to clarify what we mean by independent txs for fabric. I can remove that part if it's confusing

File: docs/vNext/Fabric_Configuration.md [hunk 2]
```diff
@@ -105,22 +91,17 @@ For the current details/documentation of the API, refer to the [source code](htt
...
-4. `timeout`: the timeout value for the transaction __in seconds__.
-
-The `invokeSettings` object has the following structure:
+The settings object has the following structure:
+* `contractId`: _string. Required._ The ID of the contract to call.
+* `contractVersion`: _string. Required._ The version of the contract to call.
 * `contractFunction`: _string. Required._ The name of the function to call in the contract.
 * `contractArguments`: _string[]. Optional._ The list of __string__ arguments to pass to the contract.
+* `readOnly`: _boolean. Optional._ Indicates whether the request is a TX or a query. Defaults to `false`.
+* `timeout`: _number. Optiona._ The timeout in seconds to use for this request.
```
[the remark this answers is not in the document]
Comment (aklenik) [reply docs/vNext/Fabric_Configuration.md]: Fixed.

File: docs/vNext/Fabric_Configuration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nklincoln) [reply docs/vNext/Fabric_Configuration.md:128]: i don't think it's an important change tbh

**Review by nklincoln — APPROVED** [review]

**Pull request merged**